projects/core/src/model/order.model.ts

```diff
@@ -22,6 +22,7 @@ export interface OrderEntry {
   deliveryMode?: DeliveryMode;
   deliveryPointOfService?: PointOfService;
   entryNumber?: number;
+  key?: string;
```

Comment:
I know we're not used to do so far, but when I make model changes today I start adding a comment.